{"version":1,"defects":{"Tests\\Unit\\RouterTest::it_registers_a_get_route":3,"Tests\\Unit\\RouterTest::it_registers_a_post_route":3,"Tests\\Unit\\RouterTest::there_are_no_routes_when_router_is_created":4,"Tests\\Unit\\RouterTest::it_resolves_route_from_a_closure":4,"Tests\\Unit\\RouterTest::it_resolves_route":4,"Tests\\Unit\\RouterTest::it_registers_a_route":3},"times":{"Tests\\Unit\\RouterTest::it_registers_a_route":0.003,"Tests\\Unit\\RouterTest::it_registers_a_get_route":0,"Tests\\Unit\\RouterTest::it_registers_a_post_route":0,"Tests\\Unit\\RouterTest::there_are_no_routes_when_router_is_created":0,"Tests\\Unit\\RouterTest::it_throws_route_not_found_exception with data set #0":0.001,"Tests\\Unit\\RouterTest::it_throws_route_not_found_exception with data set #1":0,"Tests\\Unit\\RouterTest::it_throws_route_not_found_exception with data set #2":0,"Tests\\Unit\\RouterTest::it_throws_route_not_found_exception with data set #3":0,"Tests\\Unit\\RouterTest::it_resolves_route_from_a_closure":0,"Tests\\Unit\\RouterTest::it_resolves_route":0}}